CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 * Fixed bug where reading from the `_out_` sticky local variable could return
   wrong results ([#2201](https://github.com/pry/pry/pull/2201))
+* Fixed issue with printing control characters `\001`/`^A` and `\002`/`^B`. ([#2209](https://github.com/pry/pry/pull/2209))
+  \
+  Added `Pry.config.escape_prompt` to manage adding those control characters to the prompt (the default is `true`).
+  It can then be disabled for terminals that don't have readline, or where readline's version doesn't support those characters.
+  \
+  Added support for [ConPty](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/)
```

Comment:
For what it's worth, ConPTY is *not* the name we use for the version of the console that supports VT/control sequences.

`ENABLE_VIRTUAL_TERMINAL_PROCESSING` enables VT sequences on a fairly old version of Windows 10 -- something like "10.0.10240". That's the only thing you'll need to check. :smile:

ConPTY is a newer API that lets somebody else display the UI for the console, instead of leaving us to do it. That's how somebody like alacritty or hyper can run pry! It is much newer (Windows 10 version 17763 or higher.)

Fortunately, you only need the first thing.

Reply:
(Happy to answer any questions about it! I'm the engineering lead for the console and Windows Terminal. :wave:)